ftl/common/builder_runner.py

```diff
@@ -106,7 +106,7 @@ def GenerateFTLImage(self):
             deps_image = self.GetCachedDepsImage(checksum)
             if not deps_image:
                 # TODO(aaron-prindle) make this better, prob pass args to bldr
-                if self.args.destination_path:
+                if 'destination_path' in vars(self.args):
                     deps_image = self.builder.CreatePackageBase(
```

Comment:
Maybe this would be cleaner like your TODO says, let builders declare what options they need somehow, and pass them in during construction.